packages/phoenix-event-display/src/loaders/objects/phoenix-objects.ts

```diff
@@ -586,4 +586,46 @@ export class PhoenixObjects {
 
     return object;
   }
+
+  /**
+   * Process the Fibers (a.k.a reconstructed line from points / hits) from the given parameters and get it as a geometry.
+   * @param Fiber Parameters for the SciFi Calorimeter.
+   * @returns Fiber object.
+   */
+  public static getFibers(FiberParams: any): Object3D {
```

Comment:
Since you mentioned this is only specific to LHCb, we should have a dedicated class for `LHCbObjects` just like we have for [`CMSObjects`](https://github.com/HSF/phoenix/blob/master/packages/phoenix-event-display/src/loaders/objects/cms-objects.ts). Everything works pretty much the same but in the [`CMSLoader`](https://github.com/HSF/phoenix/blob/master/packages/phoenix-event-display/src/loaders/cms-loader.ts), we call `CMSObjects.getMuonChamber` instead of `PhoenixObject.getMuonChamber`.

This guide is exactly what you are looking for: https://github.com/HSF/phoenix/blob/master/guides/developers/event-data-loader.md#handling-new-physics-objects